src/main/java/pw/kaboom/extras/modules/checks/DenyMobLag.java

```diff
@@ -0,0 +1,24 @@
+package pw.kaboom.extras.modules.checks;
+
+import org.bukkit.entity.EntityType;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.entity.CreatureSpawnEvent;
+import org.bukkit.plugin.java.JavaPlugin;
+import pw.kaboom.extras.Main;
+
+public class DenyMobLag implements Listener {
+    private static final int limit = JavaPlugin.getPlugin(Main.class).getConfig().getInt("mobLimitAtOnce");
+    private int mobSpawnCount = 0;
+
+    @EventHandler
+    public void onCreatureSpawn(CreatureSpawnEvent event) {
+        EntityType entityType = event.getEntityType();
+        if (entityType.isAlive()) {

```

Comment:
This .isAlive() check is unnecessary in a CreatureSpawnEvent listener, given that the CreatureSpawnEvent will only fire for LivingEntities.